### ISSUE TITLE
support for aggregation queries in neo4j-graphql-java

### ISSUE THREAD

**Issue by rajankit296** [neo4j-graphql/neo4j-graphql-java#290]
Currently we are not able to execute aggregation queries in neo4j-graphql-java
e.g counting number of nodes with some filter, max values

https://neo4j.com/docs/graphql-manual/current/queries/
https://neo4j.com/developer-blog/neo4j-graphql-aggregations/

**Comment by Andy2003** [comment]
This is part of the API-Alignment I'm still working on

**Comment by rajankit296** [comment]
@Andy2003 can you please suggest when this feature will include in neo4j-graphql-java.

**Comment by Andy2003** [comment]
No, since it is a really big refactoring and I have limited resources to work on this task
